deploy/helm/elastic-agent/templates/agent/k8s/daemonset.yaml

```diff
@@ -0,0 +1,31 @@
+{{- include "elasticagent.init" $ -}}
+{{- range $presetName, $presetVal := $.Values.agent.presets -}}
+{{- if and (eq ($presetVal).mode "daemonset") (eq $.Values.agent.engine "k8s") -}}
+{{- $agentName := printf "agent-%s" (lower $presetName) -}}
```

Comment:
These names should include the release name as well, imo. Otherwise, we'd get conflicts installing different agent releases in the same namespaces. What most Charts do is use the equivalent of `{{ template "elasticagent.fullname" . }}` here. That makes truncating a bit annoying, but it's necessary unless we want to explicitly forbid multiple installs in the same namespace.

This applies to every namespaced resource we name this way.

Reply:
oops, nice catch! yep this needs to change, working on it

Reply:
these names now include the helm release name as well, feel free to unresolve if you spot anything else